services/QuillLMS/client/app/bundles/Connect/components/focusPoints/focusPointsContainer.jsx

```diff
@@ -136,8 +143,20 @@ export class FocusPointsContainer extends Component {
     return <SortableList data={_.values(components)} key={_.values(components).length} sortCallback={this.sortCallback} />;
   }
 
-  renderTagsForFocusPoint = (focusPointString) => {
-    return focusPointString.split('|||').map((fp, index) => (<span className="tag is-medium is-light" key={`fp${index}`} style={{ margin: '3px', }}>{fp}</span>));
+  handleChange = (e, key) => {
+    const { focusPoints } = this.state
+    let value = e.target.value;
```

Comment:
This doesn't seem to get used - can we just delete this line and declare value as a `const` below?